javascript/builtins/SyntaxError.json

```diff
@@ -107,8 +108,8 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/1556604'>bug 1556604</a>."
+                "version_added": "103",
+                "notes": "Serialized properties: <code>name</code>, <code>message</code>, <code>cause</code>, <code>fileName</code>, <code>lineNumber</code> and <code>columnNumber</code>. Nightly only: <code>stack</code> (see <a href='https://bugzil.la/1774866'>bug 1774866</a>)."
```

Comment:
"notes": [
                  "Version 103 serialized properties: <code>name</code>, <code>message</code>, <code>cause</code>, <code>fileName</code>, <code>lineNumber</code> and <code>columnNumber</code>.",
                  "Version 104 additionally supports serialization of <code>stack</code> in <a href="https://developer.mozilla.org/docs/Web/API/Window/postMessage"><code>window.postMessage()</code></a> and <a href="https://developer.mozilla.org/docs/Web/API/structuredClone"><code>structuredClone()</code></a> ."
                ]